lib/SPIRV/OCLUtil.h

```diff
@@ -692,7 +692,7 @@ template <> inline void SPIRVMap<std::string, Op, SPIRVInstruction>::init() {
   _SPIRV_OP(isgreaterequal, FOrdGreaterThanEqual)
   _SPIRV_OP(isless, FOrdLessThan)
   _SPIRV_OP(islessequal, FOrdLessThanEqual)
-  _SPIRV_OP(islessgreater, LessOrGreater)
+  _SPIRV_OP(islessgreater, FOrdNotEqual)
```

Comment:
I'm not sure that https://github.com/KhronosGroup/SPIRV-LLVM-Translator/pull/1384/commits/cc98b0ec05162ae36ae43dd1abecf5baeb5f5296 which is created against recent changes in SPIR-V spec should be cherry-picked to old release branch. @StuartDBrady WDYT?

Reply:
I withdraw my argument. The translator is being release against llvm version and not against SPIR-V version.